Code/GraphMol/Descriptors/OxidationNumbers.h

```diff
@@ -0,0 +1,93 @@
+//
+//  Copyright (c) 2023, David Cosgrove, CozChemIx Limitied
```

Comment:
I think this whole copyright thing could be:
```
//
//  Copyright (C) 2023, David Cosgrove, CozChemIx Limited
//
//   @@ All Rights Reserved @@
//  This file is part of the RDKit.
//  The contents are covered by the terms of the BSD license
//  which is included in the file license.txt, found at the root
//  of the RDKit source tree.
//
```

We used to include the full BSD license for stuff we submitted from NIBR because the lawyers wanted to see that.